src/main/java/com/cognifide/aemrules/java/checks/resourceresolver/close/ResourceResolverTryWithResourcesCheck.java

```diff
@@ -88,8 +89,8 @@ public void visitAssignmentExpression(AssignmentExpressionTree tree) {
   @Override
   public void visitTryStatement(TryStatementTree tree) {
     insideTryStatement = true;
-    tree.resources().stream()
-        .map(resource -> resource.simpleName().name())
+    tree.resourceList().stream()
```

Comment:
Are you sure this is synonymous to the previous line? SonarCloud seems to think these lines aren't covered by unit tests. This means that they never were or that existing tests stopped following this execution path. Please investigate.

Reply:
In my opinion issue here is related to travis config. Covervall can measure coverage. I think that this issue should be moved to new one as switching to other sonar cloud account.

Reply:
There was missing configuration for Jacoco reports, it's fixed now.

Reply:
Issue for build config:
https://github.com/Cognifide/AEM-Rules-for-SonarQube/issues/183